aws/sqldatabases.bicep

```diff
@@ -50,6 +50,9 @@ param allocatedStorage string = '20'
 @description('Database license model')
 param licenseModel string = 'license-included'
 
+@description('The number of days for which automated backups are retained. Setting this parameter to a positive number enables backups. Defaults to 0.')
+param backupRetentionPeriod int = 0
```

Comment:
The default of 0 means indefinite.

Reply:
AWS CloudFormation spec says 0 should mean disabled backups: https://us-west-2.console.aws.amazon.com/cloudformation/home?region=us-west-2#/registry/activated-extensions/details/schema?arn=arn%3Aaws%3Acloudformation%3Aus-west-2%3A%3Atype%2Fresource%2FAWS-RDS-DBInstance

Reply:
Should we also add the upper and lower limits to the description?

https://docs.aws.amazon.com/sdkfornet1/latest/apidocs/html/P_Amazon_RDS_Model_ModifyDBInstanceRequest_BackupRetentionPeriod.htm